openshift/productization/generate-dockerfiles/Dockerfile.in

```diff
@@ -1,18 +1,18 @@
 FROM rhel8/go-toolset:1.12.8 AS builder
-WORKDIR /opt/app-root/src/go/src/github.com/knative/$COMPONENT
+WORKDIR /opt/app-root/src/go/src/knative.dev/$COMPONENT
 COPY . .
 RUN go build -o /tmp/$SUBCOMPONENT ./cmd/$GO_PACKAGE
 
 FROM ubi8:8-released
-COPY --from=builder /tmp/$SUBCOMPONENT /usr/bin/$SUBCOMPONENT
+COPY --from=builder /tmp/$SUBCOMPONENT /ko-app/$SUBCOMPONENT
 
 LABEL \
       com.redhat.component="openshift-serverless-1-tech-preview-$COMPONENT-$SUBCOMPONENT-rhel8-container" \
       name="openshift-serverless-1-tech-preview/$COMPONENT-$SUBCOMPONENT-rhel8" \
-      version="0.7.1" \
+      version="$VERSION" \
       summary="Red Hat OpenShift Serverless 1 $CAPITALIZED_COMPONENT $CAPITALIZED_SUBCOMPONENT" \
-      maintainer="mthoemme@redhat.com" \
+      maintainer="wvanwinc@redhat.com" \
```

Comment:
I'll hold back merging this until we have clearance on whether this shouldn't be our support email address.

Reply:
Sounds good to me. I hope to have an answer by today.